src/components/ProfileHoverCard/index.web.tsx

```diff
@@ -119,19 +144,34 @@ export function ProfileHoverCardInner(props: ProfileHoverCardProps) {
       onMouseUp={onClickTarget}>
       {props.children}
 
-      {hovered && (
+      {shouldRender && (
         <Portal>
-          <Animated.View
-            entering={FadeIn.duration(80)}
-            exiting={FadeOut.duration(80)}>
-            <div
-              ref={refs.setFloating}
-              style={floatingStyles}
-              onPointerEnter={onPointerEnterCard}
-              onPointerLeave={onPointerLeaveCard}>
-              <Card did={props.did} hide={hide} />
-            </div>
-          </Animated.View>
+          <style>
```

Comment:
Let's put this into the base HTML files inline

Reply:
Yea going to do that after figuring everything out 👍